### ISSUE TITLE
Cannot go install CLI tool

### ISSUE THREAD

**Issue by tpdownes** [GoogleCloudPlatform/cloud-foundation-toolkit#1433]
If you attempt to `go install` the CLI tool, you are presented with an error with an obvious source in cli/go.mod. Enabling me to go install the tool would simplify installation in automation environments.

```shell
$ go install github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@latest
go: downloading github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli v0.6.0
go: github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@latest (in github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@v0.6.0):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

**Comment by tpdownes** [comment]
@g-awmalik FYI

**Comment by tpdownes** [comment]
The fix from #1446 has been reverted by https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/commit/5ca3e865c81cdc9f2cafcf47b8bfb2453c1bdf01. As a consequence what once worked:

```
go install github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@v0.6.2
```

Now does not:

```
go install github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@latest
go: downloading github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli v0.7.1
go: github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@latest (in github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli@v0.7.1):
    The go.mod file for the module providing named packages contains one or
    more replace directives. It must not contain directives that would cause
    it to be interpreted differently than if it were the main module.
```

**Comment by g-awmalik** [comment]
The `go install` issue is resolved now with v0.8.0.

Regarding, the binary name being cli vs cft, we'll have to rename the package which will cause regressions. As an alternative, does "curling" the latest (or a versioned) binary from this URL: https://storage.googleapis.com/cft-cli/latest/cft, work for you?

**Comment by g-awmalik** [comment]
Closing this as renaming the package is not feasible. Alternatives are to download the binary from the GCS location or using an alias in shell.
